docs/source/tracking/tutorials/remote-server.rst

```diff
@@ -0,0 +1,260 @@
+
+ * **Part 1c and d**:
+
+  * Retrieval requests by the client return information from the configured `SQLAlchemyStore` table
+
+ * **Part 2a and b**:
+
+  * Logging events for artifacts are made by the client using the ``HttpArtifactRepository`` to write files to MLflow Tracking Server
+  * The Tracking Server then writes these files to the configured object store location with assumed role authentication
+
+ * **Part 2c and d**:
+
+  * Retrieving artifacts from the configured backend store for a user request is done with the same authorized authentication that was configured at server start
+  * Artifacts are passed to the end user through the Tracking Server through the interface of the ``HttpArtifactRepository``
+
+
+Getting Started
+===============
+
+Preface
+-------
+
+In an actual production deployment environment, you will have multiple remote hosts to run both the tracking server and databases, as shown in the diagram above. However, for the purposes of this tutorial, 
+we will just use a single machine with multiple Docker containers running on different ports, mimicking the remote environment with a far easier evaluation tutorial setup. We will also use `MinIO <https://min.io/>`_,
+an S3-compatible object storage, as an artifact store so that you don't need to have AWS account to run this tutorial.
+
+Step 1 - Get MLflow and additional dependencies
+-----------------------------------------------
+MLflow is available on PyPI. Also `pyscopg2 <https://pypi.org/project/psycopg2/>`_ and `boto3 <https://boto3.amazonaws.com/v1/documentation/api/latest/index.html>`_ are required for accessing PostgreSQL and S3 with Python.
+If you don't already have them installed on your system, you can install them with:
+
+.. code-section::
+
+    .. code-block:: bash
+        :name: install-mlflow
+
+        pip install mlflow psycopg2 boto3
+
+Step 2 - Set up remote data stores
+----------------------------------
+MLflow Tracking Server can interact with a variety of data stores to store experiment and run data as well as artifacts.
+In this tutorial, we will use **Docker Compose** to start two containers, each of them simulating remote servers in an actual environment.
+
+1. `PostgreSQL <https://www.postgresql.org/>`_ database as a backend store.
+2. `MinIO <https://min.io/>`_ server as an artifact store.
+
+Install docker and docker-compose
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+  These docker steps are only required for the tutorial purpose. MLflow itself doesn't depend on Docker at all.
+
+Follow the official instructions for installing `Docker <https://docs.docker.com/install/>`_ and `Docker Compose <https://docs.docker.com/compose/install/>`_. Then, run ``docker --version`` and ``docker-compose --version`` to make sure they are installed correctly.
+
+Create ``docker-compose.yaml``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Create a file named ``docker-compose.yaml`` with the following content:
```

Comment:
```suggestion
Create a file named ``compose.yaml`` with the following content:
```

Can we use `compose.yaml`?

https://docs.docker.com/compose/compose-file/03-compose-file/